src/systemTest/java/com/flightstats/hub/testcase/WebhookLifecycleTest.java

```diff
@@ -0,0 +1,198 @@
+                Response<String> response = call.clone().execute();
+                channelItemsPosted = parseResponse(response.body());
+                return response.code() == OK.getStatusCode()
+                        && channelItemsPosted.size() == channelItems.size();
+            });
+        } catch (Exception e) {
+            log.error("Problem verifying webhook callbacks. {} ", e.getMessage());
+            logWebhookCallbackError();
+        }
+
+        Collections.sort(channelItems);
+        Collections.sort(channelItemsPosted);
+
+        log.info("channelItemsPosted {} ", channelItemsPosted);
+        log.info("channelItems {} ", channelItems);
+
+        assertEquals(channelItems, channelItemsPosted);
+    }
+
+    private List<String> parseResponse(String body) {
+        if (!isBlank(body)) {
+            String parsedString = body.replace("[", EMPTY_STRING)
+                    .replace("]", EMPTY_STRING);
+            List<String> postedItems = Arrays.asList(parsedString.split(","));
+            postedItems.replaceAll(String::trim);
+            return postedItems;
+        }
+        return Collections.EMPTY_LIST;
+    }
+
+    @SneakyThrows
+    private void createChannel() {
+        log.info("Channel name {} ", channelName);
+
+        Call<Object> call = channelResourceClient.create(Channel.builder().name(channelName).build());
+        Response<Object> response = call.execute();
+
+        assertEquals(CREATED.getStatusCode(), response.code());
+    }
+
+    @SneakyThrows
+    private void addWebhook(Webhook webhook) {
+        log.info("Webhook name {} ", webhookName);
+
+        Call<com.flightstats.hub.model.Webhook> call = webhookResourceClient.create(webhookName, webhook);
+        Response<com.flightstats.hub.model.Webhook> response = call.execute();
+
+        assertEquals(CREATED.getStatusCode(), response.code());
+    }
+
+    private List<String> addItemsToChannel(Object data) {
+        final List<String> channelItems = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            channelItems.add(addItemToChannel(data));
+         }
+        return channelItems;
+    }
+
+    @SneakyThrows
+    private String addItemToChannel(Object data) {
+        Call<ChannelItem> call = channelItemResourceClient.add(channelName, data);
+        Response<ChannelItem> response = call.execute();
+
+        assertEquals(CREATED.getStatusCode(), response.code());
+        assertNotNull(response);
+        assertNotNull(response.body());
+
+        return response.body().get_links().getSelf().getHref();
+    }
+
+    private Webhook buildWebhook(String startItem) {
+        Webhook.WebhookBuilder builder = Webhook.builder()
+                .channelUrl(retrofit.baseUrl() + "channel/" + channelName)
+                .callbackUrl(callbackServer.getUrl())
+                .parallelCalls(2)
+                .batch("single");
+
+        if (!isBlank(startItem)) {
```

Comment:
Personal preference, but I'd just inline this with a tertiary and remove the if clause. Gets rid of the intermediate builder variable.
```
return Webhook.builder()
  .channelUrl(retrofit.baseUrl() + "channel/" + channelName)
  .callbackUrl(callbackServer.getUrl()).callbackUrl(callbackServer.getUrl())
  .parallelCalls(2)
  .batch("single") // Probably for a future day, but seems like this should be an enum
  .startItem(isBlank() ? null : startItem)
.build()
```